### DISCUSSION TITLE
drag and drop tabs and split editor pane

### DISCUSSION THREAD

**Discussion by jmls** [coder/code-server#6061] [General]
if I fire up the docker image, I can't seem to be able to drag and drop editor tabs between panes, or create a split pane by draggin to a new area

Is there a setting for this that I'm missing ?

**Comment by code-asher** [comment]
I tried with latest v4.10.1 (`docker run -p 8080:8080 codercom/code-server --auth none`) but drag and drop works for me both for moving tabs between panes and splitting panes (using Chromium for my browser) without changing any settings.

**Comment by jmls** [comment]
indeed it does ! many thanks.

**Reply by jmls** [reply]
ah, now I know why I had the issue

I had the url for the vscode server inside an ,"\<embed\>" tag. For some reason, this breaks the drag/drop. Put it in an <iframe> and it works ...

**Reply by jmls** [reply]
same issue with the "\<object\>" tag

**Reply by code-asher** [reply]
Oh interesting!  Very curious.
